src/mv/mv.rs

```diff
@@ -405,3 +423,12 @@ fn existing_backup_path(path: &PathBuf, suffix: &str) -> PathBuf {
         simple_backup_path(path, suffix)
     }
 }
+
+fn is_empty_dir(path: &PathBuf) -> bool {
+    match fs::read_dir(path) {
+        Ok(contents) => {
+            return contents.peekable().peek().is_none();
+            },
```

Comment:
Extra indent

Reply:
I'll fix this, but, in addition, I just ran a `rustfmt` against the repo, and I was going to add a PR for it.

Maybe a `cargo fmt -- --check` could be added as a CI test.

Reply:
Fixed.